site/_layouts/tech-detail.njk

```diff
@@ -32,7 +32,26 @@ layout: _wrapper
 
   <section class="tech-detail__content {% if landingPage %}tech-detail__content--landing{% endif %}" id="article">
     <div class="type">{{ content | safe }}</div>
-    {% if landingPage and landingPage.subnav %}
+    {% if landingPage and landingPage.subnav %} {% if started %}
+    <div class="tech-detail__get-started">
+      <h2 class="type--h1 tech-detail__get-started--title">{{started.title}}</h2>
+      {% if started.desc %}
+      <p class="type--large tech-detail__get-started--desc">{{started.desc}}</p>
+      {% endif %}
+      <div class="tech-detail__get-started--grid">
+        {% for card in started.cards %}
+        <div class="tech-detail__get-started--grid-item">
+          <header class="tech-detail__get-started--heading">
+            <h3 class="type--h4">{{ card.title }}</h3>
+          </header>
+          {{ card.body | markdown | safe }}
+        </div>
+        {% endfor %}
+      </div>
+    </div>
```

Comment:
can I create a data **technical.yml** to add modifiers (custom, variant)? also, for the description `started.desc` can I modify the item-grid to 
```
{% if modifiers.subtitle or desc %}
<p class="item-grid__subtitle type--large">{{modifiers.subtitle or desc}}</p>
{% endif %}
```
and if yes then I would probably have to modify the `macro itemGrid` to add desc (string) param.

These are some of the questions I had when creating this component but didn't ask you because you were busy launching the site. ;) I'm available for a quick call, if needed.

Reply:
Change my mind on item grid.